pkg/runtimeconfig/defaults.go

```diff
@@ -3,15 +3,19 @@ package runtimeconfig
 import (
 	"os"
 	"path/filepath"
+	"strings"
 
 	"github.com/gosimple/slug"
+	"github.com/replicatedhq/embedded-cluster/pkg/release"
+	kotsv1beta1 "github.com/replicatedhq/kotskinds/apis/kots/v1beta1"
 	"github.com/sirupsen/logrus"
 )
 
 // Holds the default no proxy values.
 var DefaultNoProxy = []string{"localhost", "127.0.0.1", ".cluster.local", ".svc"}
 
-const ProxyRegistryAddress = "proxy.replicated.com"
+const proxyRegistryAddress = "proxy.replicated.com"
+const replicatedAppDomain = "replicated.app"
```

Comment:
as discussed, do not hardcode this domain and instead require the license.

Reply:
I've dropped this now - the function will return empty string in that case